.lighthouserc.json

```diff
@@ -39,7 +38,7 @@
             "uses-responsive-images": ["error", {"minScore": 1}],
             "uses-optimized-images": ["error", {"minScore": 1}],
             "uses-rel-preconnect": ["error", {"minScore": 0.5}],
-            "time-to-first-byte": ["error", {"minScore": 1}],
+            "time-to-first-byte": ["off", {"minScore": 1}],
```

Comment:
Why these changes (to "off")?

Reply:
Do you mean "high" rather than "low"? I thought low values would be better.

In any case ... you probably should have a comment, or readme, or something explaining the rationale for these settings? Also, should there be a TODO here?

Reply:
Hi Sean
We can't add a comment in JSON hence we won't be able to add a TODO comment here.

Reply:
OK, so what is the plan for keeping track of this? Should an issue be filed?

Reply:
Yes, I'll create an issue for this

Reply:
Thanks -- please do that and link to it here, then this thread can be closed.

Reply:
Done https://github.com/oppia/oppia/issues/10028.